### ISSUE TITLE
Litentry-parachain error logs 

### ISSUE THREAD

**Issue by zhy827827** [litentry/litentry-parachain#2084]
Litentry-parachain version 0.9.16 
My server configuration is  CPU 16 core memory 64G disk 3T

I  deployment Litentry-parachain Full Node,
```
/home/ubuntu/litentry/litentry-collator --base-path /opt/litentrydata --name=litentry-node --chain=litentry  --trie-cache-size 0 --ws-external --rpc-external --rpc-cors=all --no-prometheus --execution=wasm --no-hardware-benchmarks  --pruning 1000 -- --execution=wasm --chain polkadot --pruning 1000
```
There are two issuse here

1. OOM KILL 
```
Aug 31 01:06:57 rockx kernel: [24182818.153057] oom-kill:constraint=CONSTRAINT_NONE,nodemask=(null),cpuset=/,mems_allowed=0,global_oom,task_memcg=/system.slice/litentry.service,task=litentry-collat,pid=3851226,uid=1000
Aug 31 01:06:57 rockx kernel: [24182818.153114] Out of memory: Killed process 3851226 (litentry-collat) total-vm:716090776kB, anon-rss:55100344kB, file-rss:0kB, shmem-rss:0kB, UID:1000 pgtables:145992kB oom_score_adj:0
```
Can't complete synchronization.OOM-KILL caused data corruption, unable to see any logs after restarting, stuck
2.After deleting the data, start to synchronize again.and error log
```
[Relaychain] 💔 The bootnode you want to connect to at `/dns/polkadot-bootnode.polkadotters.com/tcp/30333/p2p/12D3KooWCgNAXvn3spYBeieVWeZ5V5jcMha5Qq1hLMtGTcFPk93Y` provided a different peer ID `12D3KooWPAVUgBaBk6n8SztLrMk8ESByncbAfRKUdxY1nygb9zG3` than the one you expect `12D3KooWCgNAXvn3spYBeieVWeZ5V5jcMha5Qq1hLMtGTcFPk93Y`.
```

**Comment by Kailai-Wang** [comment]
Thanks for the report.

It is the first time we see this, the hardware should be more than enough to run the full node.

May I ask did you start it on any cloud instances (if so, which cloud provider?), or your own host (bare metal)?

If possible, can you try it on a different host? (Just to rule out the node instance problem)

**Comment by zhy827827** [comment]
bare metal @Kailai-Wang

**Comment by Kailai-Wang** [comment]
Is it possible that you:
1. try other machines? A cloud instance, or another bare metal
2. use the same args as in the [doc](https://docs.litentry.com/parachain/get-started/litentry-network/full-node) and see if it helps?

**Comment by zhy827827** [comment]
I have tried the same parameters as doc, and it will also be oom-kill.Let me observe this strange problem again

**Comment by zhy827827** [comment]
I restart Litentry.service, the log stays in "collating: no"

```
-- Logs begin at Tue 2023-05-16 15:21:36 UTC. --
Sep 01 01:30:22 rockx-mainnet-mix-ovh-03 sh[3874536]: 2023-09-01 01:30:22 ❤️  by Litentry Technologies GmbH <info@litentry.com>, 2017-2023
Sep 01 01:30:22 rockx-mainnet-mix-ovh-03 sh[3874536]: 2023-09-01 01:30:22 📋 Chain specification: Litentry
Sep 01 01:30:22 rockx-mainnet-mix-ovh-03 sh[3874536]: 2023-09-01 01:30:22 🏷  Node name: litentry-node
Sep 01 01:30:22 rockx-mainnet-mix-ovh-03 sh[3874536]: 2023-09-01 01:30:22 👤 Role: FULL
Sep 01 01:30:22 rockx-mainnet-mix-ovh-03 sh[3874536]: 2023-09-01 01:30:22 💾 Database: RocksDb at /opt/rockx/litentrydata/chains/litentry/db/full
Sep 01 01:30:22 rockx-mainnet-mix-ovh-03 sh[3874536]: 2023-09-01 01:30:22 ⛓  Native runtime: litentry-parachain-9168 (litentry-parachain-0.tx1.au1)
Sep 01 01:30:22 rockx-mainnet-mix-ovh-03 sh[3874536]: 2023-09-01 01:30:22 Parachain id: Id(2013)
Sep 01 01:30:22 rockx-mainnet-mix-ovh-03 sh[3874536]: 2023-09-01 01:30:22 Parachain Account: 5Ec4AhPXYUNLCp3QNT7e6bKwKz9Xx2Svdgi7egBLzMwoXqcD
Sep 01 01:30:22 rockx-mainnet-mix-ovh-03 sh[3874536]: 2023-09-01 01:30:22 Parachain genesis state: 0x000000000000000000000000000000000000000000000000000000000000000000113e8b5407e3898cc052a2c741ff1779bc5ea3e0e77394de49cab9500be4b04003170a2e7597b7b7e3d84c05391d139a62b157e78786d8c082f29dcf4c11131400
Sep 01 01:30:22 rockx-mainnet-mix-ovh-03 sh[3874536]: 2023-09-01 01:30:22 Is collating: no
```

**Comment by zhy827827** [comment]
```
Sep 04 02:28:08 rockx-mainnet-mix-ovh-03 bash[3931057]: 2023-09-04 02:28:08 [Relaychain] ⚙️  Syncing  0.0 bps, target=#17137400 (30 peers), best: #12536107 (0x0a40…15e0), finalized #12534272 (0xe794…9709), ⬇ 126.3kiB/s ⬆ 72.3kiB/s
Sep 04 02:28:10 rockx-mainnet-mix-ovh-03 bash[3931056]: /home/ubuntu/litentry/start.sh: line 18: 3931057 Killed                  /home/ubuntu/litentry/litentry-collator --base-path /opt/rockx/litentrydata --chain=litentry --no-prometheus --no-hardware-benchmarks --state-pruning=1000 --state-cache-size 0 --ws-external --rpc-external --rpc-cors=all --execution=wasm -- --execution=wasm --chain polkadot
Sep 04 02:28:10 rockx-mainnet-mix-ovh-03 systemd[1]: litentry.service: Main process exited, code=exited, status=137/n/a
Sep 04 02:28:10 rockx-mainnet-mix-ovh-03 systemd[1]: litentry.service: Failed with result 'exit-code'.
Sep 04 02:28:10 rockx-mainnet-mix-ovh-03 systemd[1]: litentry.service: Scheduled restart job, restart counter is at 15.
Sep 04 02:28:10 rockx-mainnet-mix-ovh-03 systemd[1]: Stopped Full Node.
Sep 04 02:28:10 rockx-mainnet-mix-ovh-03 systemd[1]: Started Full Node.
Sep 04 02:28:10 rockx-mainnet-mix-ovh-03 bash[3932265]: `--state-cache-size` was deprecated. Please switch to `--trie-cache-size`.

```

**Comment by zhy827827** [comment]
aways oom-kill  @Kailai-Wang

**Comment by zhy827827** [comment]
![image](https://github.com/litentry/litentry-parachain/assets/15209848/19578e3f-9df7-435c-837e-7b4ba6d02fb1)

**Comment by zhy827827** [comment]
完全跑不起来.

**Comment by zhy827827** [comment]
using docker run full node,But the same occurred OOM-KILL
https://docs.litentry.com/parachain/get-started/litentry-network/full-node#using-docker

When synchronizing to finalized blocks stage. 
```[Parachain] 💤 Idle (16 peers), best: #3132070 (0x5df2…c089), finalized #292822 (0x3268…1775),``` 
 bash[4031107]: /home/ubuntu/litentry/start.sh: line 3: 4031108 Killed     

@Kailai-Wang

**Comment by Kailai-Wang** [comment]
Thanks for the report, we'll try to run the full node ourselves and come back to you

**Comment by zhy827827** [comment]
Today, I switched to AWS server for run litentry full node, and I will provide you with feedback on the operation status

**Comment by zhy827827** [comment]
Is there a snapshot of Litentry ? @Kailai-Wang

**Comment by Kailai-Wang** [comment]
Unfortunately not :(

**Comment by Kailai-Wang** [comment]
I tried to reproduce it by using the same command and the binary from the image `litentry/litentry-parachain:v0.9.16` - It's syncing now.

- So far the RAM usage is low - but I'll keep it running for a while (like 24h) and report back later.
May I ask when did you have the OOM? At the beginning of syncing, or after certain block height?

- I also saw the message: `[Relaychain] 💔 The bootnode you want to connect to at xxx provided a different peer ID  ...`, but that seems to be a warning and the syncing continued
